ql/src/java/org/apache/hadoop/hive/ql/ddl/DDLUtils.java

```diff
@@ -231,10 +231,17 @@ public static void validateTableIsIceberg(org.apache.hadoop.hive.ql.metadata.Tab
   }
 
   public static boolean isIcebergTable(Table table) {
```

Comment:
Wow, I originally thought the change be available also for Delta Lake, Hudi, etc, but looks like we hardcode many for Iceberg...
I will create a ticket to push this kind of logic to StorageHandler...